tests/cli/test_family.py

```diff
@@ -32,9 +32,15 @@ def test_family_cutoffs_set(run_cli_command, get_pseudo_family, generate_cutoffs
     assert "Error: Missing option '-s' / '--stringency'" in result.output
     assert sorted(family.get_cutoff_stringencies()) == sorted(['low', 'normal'])
 
-    # Invalid cutoffs structure
+    # Missing cutoffs
     filepath.write_text(json.dumps({'Ar': {'cutoff_rho': 300}}))
     result = run_cli_command(cmd_family_cutoffs_set, [family.label, str(filepath), '-s', 'high'], raises=True)
+    assert 'is missing cutoffs for element:' in result.output
+    assert sorted(family.get_cutoff_stringencies()) == sorted(['low', 'normal'])
+
+    # Invalid cutoffs structure
+    filepath.write_text(json.dumps({'Ar': {'cutoff_rho': 300, 'cutoff_wfc': 300, 'GME': 'moon'}}))
```

Comment:
I am confused by this test. I thought the whole idea of one of the commits was to allow for extraneous keys, like `GME` in this example, and not have the command except because of it, but simply ignore it. So what is going on here? Why should this command fail?

Reply:
Yeah, seems the command was actually failing for a different reason (i.e. not all elements being present). This has now been fixed in https://github.com/aiidateam/aiida-pseudo/pull/132